src/app/core/services/notifications/notifications.service.ts

```diff
@@ -0,0 +1,152 @@
+import { Injectable } from '@angular/core';
+import { MatSnackBar } from '@angular/material';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class NotificationsService {
+  constructor(public snackbar: MatSnackBar) {}
+
+  openSavedSnackBar() {
```

Comment:
All of these methods look very similar, could they be consolidated into one method that accepts a message and class parameter? Even further, could the snackbars be configured globally to where you don't need to specify duration and horizontalPosition?

Reply:
Ok the changes have been committed